e2e_samples/parking_sensors/scripts/configure_databricks.sh

```diff
@@ -54,6 +55,42 @@ databricks workspace import "$databricks_folder_name/01_explore.py" --file "./da
 databricks workspace import "$databricks_folder_name/02_standardize.py" --file "./databricks/notebooks/02_standardize.py" --format SOURCE --language PYTHON --overwrite
 databricks workspace import "$databricks_folder_name/03_transform.py" --file "./databricks/notebooks/03_transform.py" --format SOURCE --language PYTHON --overwrite
 
+# Define suitable VM for DB cluster
+file_path="./databricks/config/cluster.config.json"
+
+# Get available VM sizes in the specified region
+vm_sizes=$(az vm list-sizes --location "$AZURE_LOCATION" --output json)
+
+# Get available Databricks node types using the list-node-types API
+node_types=$(databricks clusters list-node-types --output json)
+
+# Extract VM names and node type IDs into temporary files
+echo "$vm_sizes" | jq -r '.[] | .name' > vm_names.txt
+echo "$node_types" | jq -r '.node_types[].node_type_id' > node_type_ids.txt
+
+# Find common VM sizes
+common_vms=$(grep -Ff node_type_ids.txt vm_names.txt)
+
+# Find the VM with the least resources
+least_resource_vm=$(echo "$vm_sizes" | jq --arg common_vms "$common_vms" '
+  map(select(.name == ($common_vms | split("\n")[]))) |
+  sort_by(.numberOfCores, .memoryInMb) |
```

Comment:
there is a typo, so it currenrtly does not sort by MB (because the field is `memoryInMB` and not `memoryInMb`), with the below suggestion it will also sort by MB
```suggestion
  sort_by(.numberOfCores, .memoryInMB) |
```